repository-data/application/src/main/resources/hcm-content/content/documents/lks/guidance-pages.yaml

```diff
@@ -0,0 +1,444 @@
+/content/documents/lks/guidance-pages:
```

Comment:
Is there a better place to manage dev content ? Perhaps `repository-data/development` & `repository-data/site-development` ?

Reply:
@smanivasag Based on the definition from this document https://documentation.bloomreach.com/14/library/concepts/content-repository/content-bootstrap.html we should move this bootstrap content into `repository-data/development`